operator/option/config.go

```diff
@@ -324,8 +308,6 @@ func (c *OperatorConfig) Populate() {
 	c.UnmanagedPodWatcherInterval = viper.GetInt(UnmanagedPodWatcherInterval)
 	c.NodeCIDRMaskSizeIPv4 = viper.GetInt(NodeCIDRMaskSizeIPv4)
 	c.NodeCIDRMaskSizeIPv6 = viper.GetInt(NodeCIDRMaskSizeIPv6)
-	c.IPAMOperatorV4CIDR = viper.GetStringSlice(IPAMOperatorV4CIDR)
-	c.IPAMOperatorV6CIDR = viper.GetStringSlice(IPAMOperatorV6CIDR)
```

Comment:
For the operator, are these superseded by the calls in `DaemonConfig`? I'm not so familiar with this code but basically what I want to know is that we're not breaking this option for the operator. How do we establish this?